plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/ExplodedWarPublisher.java

```diff
@@ -10,12 +13,10 @@
 import org.eclipse.wst.common.componentcore.ComponentCore;
 import org.eclipse.wst.server.core.util.PublishHelper;
 
-import com.google.cloud.tools.eclipse.appengine.deploy.Messages;
-import com.google.common.base.Preconditions;
-
 /**
  * Writes the exploded WAR file of a project to a staging directory.
  */
+@SuppressWarnings("restriction")
```

Comment:
restriction warnings I'm tempted to leave in. It feels like we're taking some risks with this code, and I'd like to figure out exactly how much.